components/climate/tuya.rst

```diff
@@ -37,10 +47,32 @@ Based on this, you can create the climate device as follows:
       - platform: tuya
         name: "My Climate Device"
         switch_datapoint: 1
-        target_temperature_datapoint: 3
-        current_temperature_datapoint: 4
-        eco_datapoint: 7
-        eco_temperature: 20 °C
+        target_temperature_datapoint: 2
+        current_temperature_datapoint: 3
+        supports_heat: True
+        supports_cool: True
+        active_state:
+          datapoint: 4
+          cooling_value: 0
+          heating_value: 1
+          drying_value: 2
+          fanonly_value: 3
+        preset:
+          eco:
+            datapoint: 8
+            temperature: 28
+          sleep:
+            datapoint: 101
+        swing_mode:
+          vertical_datapoint: 106
+          horizontal_datapoint: 107
+        fan_mode:
+          datapoint: 5
+          auto_value: 0
+          low_value: 2
+          medium_value: 3
+          middle_value: 4
+          high_value: 1
```

Comment:
Same as above, except for the breaking change parts

Reply:
I think I got it in the right format this time.